From db00c8c22dfbfdafb188f4e8796163af9d7fda59 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 18 Jun 2025 07:35:40 +0200
Subject: [PATCH] Stop installing `examples` into `site-packages`

Fix `setup.py` not to install `examples` as a top-level package
into `site-packages`.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 70162daa..a35de99b 100755
--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ def fread(fn):
     url='https://github.com/oauthlib/oauthlib',
     platforms='any',
     license='BSD-3-Clause',
-    packages=find_packages(exclude=('docs', 'tests', 'tests.*')),
+    packages=find_packages(exclude=('docs', 'examples', 'tests', 'tests.*')),
     python_requires='>=3.8',
     extras_require={
         'rsa': rsa_require,
